force-app/main/default/classes/SOQL.cls

```diff
@@ -248,7 +263,9 @@ public with sharing class SOQL {
      * @return The Account Name of the Contact.
      */
     public static String getAccountNameForContact(String contactId) {
-        return null; // Replace null with the variable you used to store the result
+        List<Contact> accountWithContactId = [SELECT Id, Name, Account.Name, Account.Id FROM Contact WHERE Id = :contactId];
```

Comment:
If you are filtering by the Id you can also just return the query into a single object.